test/e2e/egressip.go

```diff
@@ -1037,19 +1037,27 @@ spec:
 
 		ginkgo.By("7. Check the OVN DB to ensure no SNATs are added for the standby egressIP")
 		dbPods, err := framework.RunKubectl("ovn-kubernetes", "get", "pods", "-l", "name=ovnkube-db", "-o=jsonpath='{.items..metadata.name}'")
+		dbContainerName := "nb-ovsdb"
+		if isInterconnectEnabled() {
```

Comment:
Assuming Numan keeps the container name the same for the db, independent of `isInterconnetEnabled`, we will not need this, right?

Reply:
right. we will still need to pick up that specific node we care about but container names should not change.